ldes-server-pagination/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/pagination/batch/PaginationJobDefinitions.java

```diff
@@ -46,6 +54,13 @@ public Step paginationStep(JobRepository jobRepository, PlatformTransactionManag
 				)
 				.allowStartIfComplete(true)
 				.taskExecutor(taskExecutor)
+				.listener(new StepExecutionListener() {
+					@Override
+					public ExitStatus afterStep(@NotNull StepExecution stepExecution) {
+						serverMetrics.updatePaginationCounts(stepExecution.getJobParameters().getString(COLLECTION_NAME));
+						return StepExecutionListener.super.afterStep(stepExecution);
+					}
```

Comment:
Just so that I stay consistent as reviewer: personally, I would write a seperate class for this 🙃